README.md

```diff
@@ -73,152 +73,62 @@ To use Jazzer.js in your own project follow these few simple steps:
 
 ## Usage
 
-### Creating a fuzz target
+Jazzer.js can be used in two ways: Creating dedicated fuzz targets, as shown in
+the `Quickstart` section, or integrated into the Jest test framework.
 
-Jazzer.js requires an entry point for the fuzzer, this is commonly referred to
-as fuzz target. A simple example is shown below.
+### Using test framework integration
 
-```js
-module.exports.fuzz = function (data) {
-	myAwesomeCode(data.toString());
-};
-```
-
-A fuzz target module needs to export a function called `fuzz`, which takes a
-`Buffer` parameter and executes the actual code under test.
-
-The `Buffer`, a subclass of `Uint8Array`, can be used to create needed
-parameters for the actual code under test, so that the fuzzer can detect the
-usage of parts of the input and mutate them in the next iterations to reach new
-code paths. However, `Buffer` is not the nicest abstraction to work with. For
-that reason, Jazzer.js provides a wrapper class `FuzzedDataProvider` that allows
-reading primitive types from the `Buffer`. An example on how to use the `data`
-and the `FuzzedDataProvider` class is shown below.
-
-```js
-const { FuzzedDataProvider } = require("@jazzer.js/core");
-
-module.exports.fuzz = function (fuzzerInputData) {
-	const data = new FuzzedDataProvider(fuzzerInputData);
-	const intParam = data.consumeIntegral(4);
-	const stringParam = data.consumeString(4, "utf-8");
-	myAwesomeCode(intParam, stringParam);
-};
-```
-
-For more information on how to use the `FuzzedDataProvider` class, please refer
-to the [example](./examples/FuzzedDataProvider/fuzz.js), the
-[documentation](./packages/core/FuzzedDataProvider.ts) of the
-`FuzzedDataProvider` class, and the
-[tests](./packages/core/FuzzedDataProvider.test.ts).
-
-#### Asynchronous fuzz targets
-
-Jazzer.js supports asynchronous fuzz targets out of the box, no special handling
-or configuration is needed.
-
-The resolution of a `Promise` returned by a fuzz target is awaited before the
-next fuzzing input is provided. This enables the fuzzing of `async`/`await`,
-`Promise` and callback based code.
-
-Asynchronous code needs careful synchronization between the
-[Node.js Event Loop](https://nodejs.org/en/docs/guides/event-loop-timers-and-nexttick/)
-and the fuzzing thread, hence provides a lower throughput compared to
-synchronous fuzzing. Even so, asynchronous fuzzing is the default mode of
-Jazzer.js due to its prevalence in the JavaScript ecosystem and because it works
-for all fuzz targets.
-
-Solely synchronous code can participate in the enhanced performance of
-synchronous fuzzing by setting the `--sync` flag when starting the fuzzer.
+To use fuzzing in your normal development workflow a tight integration into the
+[Jest test framework](https://jestjs.io/) is provided. Using this, fuzz tests
```

Comment:
~~Using this~~ This coupling allows the execution of fuzz tests alongside...?

Reply:
done